lib/src/tooltip/types.tsx

```diff
@@ -0,0 +1,18 @@
+type Position = "bottom" | "top" | "left" | "right";
+
+type Props = {
+  /**
+   * Preferred position for displaying the tooltip. It may adjust automatically based on available space.
+   */
+  position?: Position;
```

Comment:
This is not a very complex type. I think it is better for the users to have the possible strings here instead ot a new type.